.gitignore

```diff
@@ -1,3 +1,8 @@
+# Byte-compiled / optimized / DLL files
+__pycache__/
+*.py[cod]
```

Comment:
`.pyc` is separately listed below already, the line `*.pyc` should be removed.

Reply:
Done in https://github.com/metabrainz/picard/pull/2625/commits/69ce41b518c05b5133ea659b328d270aa2b83857